### ISSUE TITLE
[EPIC] Data Security & Privacy Improvements

### ISSUE THREAD

**Issue by wes-otf** [HyphaApp/hypha#3712]
*This is a living epic that will grow over time*

## Description

The main goal of this epic is to ensure Hypha is logically retaining data and abides by OTF's data security/privacy policies.

## Users

- [ ] Handling of user deletion #3441 
- [x] #2192

**Comment by frjo** [comment]
Cleaning on input and not output would be a nice improvement here I think.

I'm really attracted to the idea of converting all input into markdown and only saving that instead of the HTML spagetti we are saving now. Then no more need for things like the TinyMCE editor or similar.

**Comment by wes-otf** [comment]
@frjo I totally agree! That would probably end up also being an item of #3713 as I think it would save us quite a bit of time in the future with features like compare
